taxi/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        to=Manufacturer, on_delete=models.CASCADE, related_name="manufacturers"
```

Comment:
remind yourself how to name `related_name` properly
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

Reply:
I saw that you requested changes from Yaroslav with the same problem. But it was too late xD
![image](https://user-images.githubusercontent.com/25363499/199186621-39bc4fe7-03d8-4272-a35e-b7c5e0ee82a4.png)